Dockerfile

```diff
@@ -1,25 +1,41 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 MAINTAINER Pavlina Bortlova <pbortlov@redhat.com>
 
-LABEL description="Review-rot - gather information about opened merge or pull requests"
-LABEL summary="review-rot git gitlab github pagure gerrit"
-LABEL vendor="PnT DevOps Automation - Red Hat, Inc."
+LABEL \
+    description="Review-rot - gather information about opened merge or pull requests" \
+    summary="review-rot git gitlab github pagure gerrit" \
+    vendor="EXD Process-tools guild <exd-guild-process-tools@redhat.com>"
 
 USER root
 
-RUN yum install -y epel-release && yum update -y && \
-    yum install -y git gcc python-devel \
-    python-setuptools python-pip libyaml-devel && \
-    yum clean all
+RUN microdnf update \
+    && microdnf install \
+        gcc \
+        git \
+        libyaml-devel \
+        python3-devel \
+        python3-pip \
+        python3-setuptools \
+    && microdnf clean all
 
 # copy workdir for installation of review-rot
-WORKDIR /reviewrot
-ADD . /reviewrot
+COPY . /reviewrot
 
 # install review-rot
-RUN pip install --upgrade pip setuptools && python setup.py install
+RUN cd /reviewrot \
+    && pip3 install --upgrade pip setuptools \
+    && python3 setup.py install
+
+# clean up
```

Comment:
Why do we clean up the installed libs? Could you add a comment?

Reply:
Comment updated.